build_runner/bin/build_runner.dart

```diff
@@ -23,6 +25,16 @@ Future<Null> main(List<String> args) async {
     ..addCommand(new _GenerateBuildScript());
   var parsedArgs = commandRunner.parse(args);
   var commandName = parsedArgs.command?.name;
+
+  if (parsedArgs.rest.isNotEmpty) {
```

Comment:
we detect this today with `commandName == null` - should we continue to do that and remove that condition below?

Reply:
`null` is no command. I think we still want to print help and exit code 0 there.